braket_features/Getting devices and checking device properties.ipynb

```diff
@@ -0,0 +1,340 @@
+   "source": [
+    "# specify a device directly by device arn\n",
+    "# Rigetti\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/rigetti/Aspen-8\")\n",
+    "# IonQ\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/ionq/ionQdevice\")\n",
+    "# DWave\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6\")\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/d-wave/Advantage_system1\")\n",
+    "# the cloud simulator\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/quantum-simulator/amazon/sv1\")\n",
+    "# the local simulator\n",
+    "device = LocalSimulator()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Properties of devices\n",
+    "\n",
+    "The properties of a device can be checked with the `device.properties` call. We show the most useful properties one can check for each device with the following examples. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The supported operations of SV1 include ['ccnot', 'cnot', 'cphaseshift', 'cphaseshift00', 'cphaseshift01', 'cphaseshift10', 'cswap', 'cy', 'cz', 'h', 'i', 'iswap', 'pswap', 'phaseshift', 'rx', 'ry', 'rz', 's', 'si', 'swap', 't', 'ti', 'unitary', 'v', 'vi', 'x', 'xx', 'xy', 'y', 'yy', 'z', 'zz']\n",
+      "\n",
+      " The supported result types include [ResultType(name='Sample', observables=['x', 'y', 'z', 'h', 'i', 'hermitian'], minShots=1, maxShots=100000), ResultType(name='Expectation', observables=['x', 'y', 'z', 'h', 'i', 'hermitian'], minShots=0, maxShots=100000), ResultType(name='Variance', observables=['x', 'y', 'z', 'h', 'i', 'hermitian'], minShots=0, maxShots=100000), ResultType(name='Probability', observables=None, minShots=1, maxShots=100000), ResultType(name='Amplitude', observables=None, minShots=0, maxShots=0)]\n",
+      "\n",
+      " The maximum number of qubits supported by this device is 34\n",
+      "The shots range of this device is (0, 100000)\n",
+      "The price of running tasks on this device: price=4.5 unit='hour'\n"
+     ]
+    }
+   ],
+   "source": [
+    "# the local simulator\n",
+    "device = LocalSimulator()\n",
+    "# the cloud-based simulator\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/quantum-simulator/amazon/sv1\")\n",
+    "\n",
+    "support_gates = device.properties.action['braket.ir.jaqcd.program'].supportedOperations\n",
+    "support_result_types = device.properties.action['braket.ir.jaqcd.program'].supportedResultTypes\n",
+    "qubit_count = device.properties.paradigm.qubitCount\n",
+    "shorts_range = device.properties.service.shotsRange\n",
+    "device_cost = device.properties.service.deviceCost\n",
+    "\n",
+    "print(f'The supported operations of {device.name} include {support_gates}')\n",
+    "print('\\n The supported result types include', support_result_types)\n",
```

Comment:
Remove space after first \n. Ditto below.

Reply:
there is no space in the original ipynb file

Reply:
I do see a space in the notebook you sent me.